spec/test_setup_specs/test_setup_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rspec'
+
+describe 'TestSetupSpec' do
+  before(:all) do
+    @test_setup = TestSetup.new
+  end
+
+  it 'finds trait pre-defined in test setup [:primary, :unpublished]' do
+    course = @test_setup.get(:course, :primary, :unpublished)
+
+    expect(course).to eq(Course.first)
```

Comment:
Will this survive as we pre-cache more variations of course?

Reply:
I think maybe we should just test that it does return a course and that it has the right properties

Reply:
Yeah agree 👍